routes/api/circuits.js

```diff
@@ -0,0 +1,15 @@
+const { F1 } = require('../../constants/championship');
+const { Circuit } = require('../../models');
+const modelRestRouterFactory = require('../../utils/modelRestRouterFactory');
+const jwtAuth = require('../../plugins/express/jwt-auth');
+const user = require('../../plugins/express/user');
+const isAdmin = require('../../plugins/express/isAdmin');
+
+module.exports = modelRestRouterFactory({
+  Model: Circuit,
+  writeMiddlewares: [jwtAuth, user, isAdmin],
```

Comment:
(very very opt) I would extract all these writeMiddlewares to a constants or similar, and add in all entities that you are using. 
`const writeMiddlewaresDefault = [jwAuth, user, isAdmin]`

and all writeMiddlewares:
`writeMiddlewares: [...writeMiddleWares]`